### PR TITLE
Fixes the trash bag not being able to store kevinz000

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
fix: The trash bag can now correctly store all trash.
/:cl:

### CODE REVIEW

**Comment by lzimann** [comment]
you mispelled xhuis in the ckey check

**Comment by ninjanomnom** [comment]
What if just all catpeople could be put in the trash, would that be better @lzimann

**Comment by ShizCalev** [comment]
doesn't even compile

**Comment by Hatterhat** [comment]
kev deserves to be in the trash, xhuis is good @lzimann

**Comment by Xhuis** [comment]
I was making an edit but lzimann pounced on the opportunity to close a PR of mine

**Comment by Hatterhat** [comment]
>implying the maintainers do anything good

File: code/game/objects/items/storage/bags.dm [hunk 1]
```diff
@@ -42,6 +42,14 @@
 	can_hold = list() // any
 	cant_hold = list(/obj/item/disk/nuclear)
 
+/obj/item/storage/bag/trash/attack(mob/living/user, mob/living/target)
+	if(target.key == "kevinz000")
+		user.visible_message("<span class='notice'>[user] scoops up the catboy trash!</span>", "<span class='notice'>Eww... this is some really disgusting trash.</span>")
+		target.death()
+		target.forceMove9src)
```
[the remark this answers is not in the document]
Comment (Xhuis) [reply code/game/objects/items/storage/bags.dm]: no

File: code/game/objects/items/storage/bags.dm [hunk 2]
```diff
@@ -42,6 +42,14 @@
 	can_hold = list() // any
 	cant_hold = list(/obj/item/disk/nuclear)
 
+/obj/item/storage/bag/trash/attack(mob/living/user, mob/living/target)
+	if(target.key == "kevinz000")
```
[the remark this answers is not in the document]
Comment (Xhuis) [reply code/game/objects/items/storage/bags.dm]: you're not my dad

File: code/game/objects/items/storage/bags.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (duncathan) [reply code/game/objects/items/storage/bags.dm]: gitbanned

**Pull request closed without merging**